README.md

```diff
@@ -1,2 +1,359 @@
 # ovirt-security-groups-demo
 Ansible roles and tools to demo networking API security groups in oVirt
+
+## Motivation and goals
+Configure fine grained access control to / from the oVirt VMs attached to
+external networks in OVN. Configuring access to those VMs is based on:
+* Ports
+* Protocols
+* Remote IPs
+* group membership
+* ether type
+...
+
+In the networking API, defined in OpenStack by Neutron, there is an entity to
+enable these use cases, through [security groups](https://developer.openstack.org/api-ref/network/v2/#security-groups-security-groups).
+Using security groups, the user could - for instance - configure ICMP access to
+all the VMs in OVN (or a subset of VMs), limit the traffic in the OVN external
+networks to web traffic (e.g. tcp port 80). More complex scenarios can be
```

Comment:
Right you are!

I saw that I stumbled upon this quite a lot in the remaining doc ... Is selective dyslexity a thing ? (is dyslexity even a word ? ... )

Reply:
I also used to use e.g. and i.e. interchangeably, and most people do as well. But since i read an article about it, i cannot use e.g. for everything anymore. It is like a curse.
Now i moved this curse to you. Good luck >:)